### PR TITLE
Add support for security manager endpoints

### LINKED ISSUE #2529: Support for organization security managers
I'm planning to add this support, using this issue to track.

- docs: [Security manager role](https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/managing-security-managers-in-your-organization)
- docs: [API](https://docs.github.com/en/rest/orgs/security-managers)

### PR DESCRIPTION
**Pull request by jeremyhayes**
Fixes #2529

### CODE REVIEW

File: github/orgs_security_managers.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
+// Copyright 2013 The go-github AUTHORS. All rights reserved.
```
Comment (gmlewis) [inline github/orgs_security_managers.go:1]:
```suggestion
// Copyright 2022 The go-github AUTHORS. All rights reserved.
```

File: github/orgs_security_managers.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+		return nil, resp, err
+	}
+
+	return teams, resp, nil
+}
+
+// AddSecurityManagerTeam adds a team to the list of security managers for an organization.
+//
+// GitHub API docs: https://docs.github.com/en/rest/orgs/security-managers#add-a-security-manager-team
+func (s *OrganizationsService) AddSecurityManagerTeam(ctx context.Context, org string, team string) (*Response, error) {
```
Comment (gmlewis) [inline github/orgs_security_managers.go:36]:
```suggestion
func (s *OrganizationsService) AddSecurityManagerTeam(ctx context.Context, org, team string) (*Response, error) {
```

File: github/orgs_security_managers.go [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+		return nil, err
+	}
+
+	return s.client.Do(ctx, req, nil)
+}
+
+// RemoveSecurityManagerTeam removes a team from the list of security managers for an organization.
+//
+// GitHub API docs: https://docs.github.com/en/rest/orgs/security-managers#remove-a-security-manager-team
+func (s *OrganizationsService) RemoveSecurityManagerTeam(ctx context.Context, org string, team string) (*Response, error) {
```
Comment (gmlewis) [inline github/orgs_security_managers.go:49]:
```suggestion
func (s *OrganizationsService) RemoveSecurityManagerTeam(ctx context.Context, org, team string) (*Response, error) {
```

File: github/orgs_security_managers_test.go [hunk 4]
```diff
@@ -0,0 +1,144 @@
+// Copyright 2013 The go-github AUTHORS. All rights reserved.
```
Comment (gmlewis) [inline github/orgs_security_managers_test.go:1]:
```suggestion
// Copyright 2022 The go-github AUTHORS. All rights reserved.
```

File: github/orgs_security_managers_test.go [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+
+	testNewRequestAndDoFailure(t, methodName, client, func() (*Response, error) {
+		got, resp, err := client.Organizations.ListSecurityManagerTeams(ctx, "o")
+		if got != nil {
+			t.Errorf("testNewRequestAndDoFailure %v = %#v, want nil", methodName, got)
+		}
+		return resp, err
+	})
+}
+func TestOrganizationsService_ListSecurityManagerTeams_invalidOrg(t *testing.T) {
```
Comment (gmlewis) [inline github/orgs_security_managers_test.go:51]: Please add a blank line above this.  I'm surprised `gofmt` didn't catch this.
Please make sure to run `gofmt` as described in CONTRIBUTING.md.

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @jeremyhayes !
Just a few minor tweaks please.

File: github/orgs_security_managers_test.go (hunk 5 above)
Comment (jeremyhayes) [reply github/orgs_security_managers_test.go:52]: Yeah, weird. Ran it multiple times to check. 🤷 

Newline added.

**Review by gmlewis — APPROVED** [review]
Thank you, @jeremyhayes !
LGTM.

Awaiting second LGTM+Approval from any other contributor to this repo before merging.

**Review by valbeat — APPROVED** [review]
LGTM!

**Review by raynigon — APPROVED** [review]
LGTM

**Comment by gmlewis** [comment]
Thank you, @valbeat and @raynigon !
Merging.

**Pull request merged**